services/wallet/api.go

```diff
@@ -60,6 +60,157 @@ func (api *API) GetTransfersByAddress(ctx context.Context, address common.Addres
 	return castToTransferViews(rst), nil
 }
 
+type StatsView struct {
+	BlocksStats    map[int64]int64 `json:"blocksStats"`
+	TransfersCount int64           `json:"transfersCount"`
+}
+
+// GetTransfersStats
+func (api *API) GetTransfersStats(ctx context.Context, address common.Address) (*StatsView, error) {
+	blocksStats, transfersCount := api.s.db.GetTransfersStats(address)
```

Comment:
Needs error checking (after implementing passing error up from these calls.)

Reply:
I'm going to remove this method, this was rather for testing purposes, it is useless otherwise